tests/lib/tools/snaps-state

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
```

Comment:
Please use `-e` mode as otherwise none of the error handling will bubble the error to the caller.

Reply:
done

Reply:
done